### ISSUE TITLE
Tessellation tried but got WrongShaderType

### ISSUE THREAD

**Issue by greenmughal** [vulkano-rs/vulkano-examples#16]
Tessellation tried but got WrongShaderType?

**Comment by tomaka** [comment]
Hi. Could you rephrase and give more details? I don't have any idea what you tried.

**Comment by greenmughal** [comment]
Hi, I will respond in after 10 hours since im out of home. Thanks

**Comment by greenmughal** [comment]
Please review this simple passthrough tessellation shader and setup.

https://pastebin.com/XnyuZzV4
